user-guide/Advanced_Functionality/Security/About_DMS_Security/Cloud_connectivity_and_security.md

```diff
@@ -0,0 +1,44 @@
+---
+uid: Cloud_connectivity_and_security
+---
+
+# Cloud connectivity and security
+
+From version 10.2.0 onwards, DataMiner comes bundled with a load of cloud features. Because the cloud means connecting your system to the dangers of the internet, we built security into the core design of our Cloud Connectivity framework.
+
+In highly secured environments, it is not always possible to set up a direct internet connection. For those scenarios, we intend to add support for hosting a dedicated Cloud Gateway node in a DMZ, acting as a tunnel to the DataMiner Cloud Platform.
+
+In any case, connecting your DataMiner System to our Cloud Platform is opt-in, meaning DataMiner will not transfer anything to the cloud without an Administrator explicitly enabling this on the DataMiner System.
+
+## Connecting to the DataMiner Cloud Platform
+
+To connect your DataMiner System to the DataMiner Cloud Platform, please refer to [Connecting your DataMiner System to the cloud](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+The DataMiner Cloud Platform is hosted in Microsoft Azure on the following endpoints:
+
+- ``https://*.dataminer.services/``
+- ``wss://tunnel.dataminer.services/``
+
+Only TLS-encrypted connections are allowed. Currently, only TLS 1.2 is supported. During the initial setup of a cloud connection, the DCP provides the DMS with authentication credentials (JSON Web Tokens). The DMS then uses these credentials to authenticate towards the cloud for all subsequent sessions. Every JSON web token is only valid for 1 week, after which it is automatically renewed using a single-use refresh key.
+
+It is important to note that users can disconnect their system from the DataMiner Cloud Platform at any given time. For more information, please refer to [Connecting your DataMiner System to the cloud](xref:Connecting_your_DataMiner_System_to_the_cloud).
+
+For further technical details, please refer to [Cloud Connection Security](xref:CloudConnectionSecurity).
+
+## Live Sharing
+
+Users can share a dashboard by clicking the *Start sharing* button in the dashboard menu bar. They can then specify a list of email addresses to share the dashboard with. Those email addresses will receive a link to the shared dashboard.
+
+When the recipients open the link, they will need to authenticate by logging in with the account on the DataMiner Cloud Platform linked to that email address, or by creating a dedicated DataMiner Cloud Platform account if they don't have any of the formerly mentioned accounts. After successfully authenticating, the recipient will be directed to the dashboard through the DataMiner Cloud Platform tunnel.
+
+No data from the private DataMiner System hosting the dashboard is stored on the DataMiner Cloud Platform. The share can be deleted in the same way as it was created. All emailed links will then automatically become unreachable. For more information, please refer to [Sharing a dashboard](xref:Sharing_a_dashboard).
```

Comment:
Can you add a "see also" link on the "Sharing a dashboard" page (under Sharing a dashboard using the Live Sharing Service) back to this page?